cms_mrp_bom_form/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+CMS MRP BoM Form
+================
+
+This module allows website visitors to search and browse through the list of
+Bills of Materials that the company has decided to publish.
+
+
+Usage
+=====
+
+Go on the website app of Odoo. Then you can reach the following pages:
+
+* /mrp_boms: shows you a nice form to search and list Bills of Materials.
```

Comment:
Do you foresee other records from mrp area that you might want to publish? 
If yes, then I'd prefix all the routes, like: `/mrp/boms`.